### DISCUSSION TITLE
How to participate in Community❓

### DISCUSSION THREAD

**Discussion by o1lab** [nocodb/nocodb#8834] [General]
# Welcome
Thank you for considering to contribute to NocoDB 

# Participating 
- Have a look at [community](https://github.com/nocodb/nocodb/issues?q=is%3Aopen+is%3Aissue+label%3A%22%F0%9F%91%8B+For+%3A+Community+or+Good+First+Issue%22) tagged issues.
- Ping noco-team in [discord](https://discord.gg/5RgZmkW) to discuss and we will assign the issue to you with any heads up on it.

# Contributing
- How to run [locally for development](https://docs.nocodb.com/engineering/development-setup/)
- Please read [NocoDB Contributing Guide](https://github.com/nocodb/nocodb/blob/master/.github/CONTRIBUTING.md)

Happy hacking

**Comment by imknight** [comment]
suggestion, write nocodedb related blog post and can submit as PR for hacktoberfest ?

**Comment by ptman** [comment]
hacktoberfest is over
